var/spack/repos/builtin/packages/hypre/package.py

```diff
@@ -110,8 +110,20 @@ def install(self, spec, prefix):
                         '-rhsone')
             make("install")
 
+    @property
+    def headers(self):
+        """Export the main hypre header, HYPRE.h; all other headers can be found
+        in the same directory.
+        Sample usage: spec['hypre'].headers.cpp_flags
+        """
+        hdrs = HeaderList(find(self.prefix.include, 'HYPRE.h', recursive=False))
+        return hdrs or None
+
     @property
     def libs(self):
-        is_shared = self.spec.satisfies('+shared')
-        return find_libraries('libHYPRE', root=self.prefix,
-                              shared=is_shared, recursive=True)
+        """Export the hypre library.
+        Sample usage: spec['hypre'].libs.ld_flags
+        """
+        libs = find_libraries('libHYPRE', root=self.prefix.lib,
```

Comment:
Are they always in ‘lib’ and not ‘lib64’ on some platforms?

Reply:
I'll add fallbacks to search `prefix.lib64` and as a last resort all of `prefix`.